### PR TITLE
adds a ghost transformation syringe to the abandoned ghost station

### PR DESCRIPTION
**Pull request by boodaliboo**
ruin is shit this makes it not shit

#### Changelog

:cl:  
rscadd: Added ghost syringe to ghost station
/:cl:

### CODE REVIEW

**Comment by SomeguyManperson** [comment]
lets you turn into a ghost and see/hear ghosts so uh no

**Comment by boodaliboo** [comment]
give me ideas for a different item

**Comment by deathslayer2481** [comment]
Ghost sword

**Review by patpol4 — CHANGES_REQUESTED** [review]
Neither of those fit in, and are too powerful, if you want to add more loot to ruins, add something that makes sense. Not every ruin has to have powergamer loot.

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]
ye massive no, close this if you can't think of anything else

**Comment by Ratvar** [comment]
Make people who enter station become a ghost forever

**Comment by patpol4** [comment]
This has now become a code PR, transferring control over to codermans

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 1]
```diff
@@ -260,6 +260,13 @@
 	amount_per_transfer_from_this = 1
 	volume = 1
 	list_reagents = list(/datum/reagent/gluttonytoxin = 1)
+	
+/obj/item/reagent_containers/syringe/ghost
+	name = "Gluttony's Blessing"
```
Comment (patpol4) [inline code/modules/reagents/reagent_containers/syringes.dm]: gluttony's blessing?

File: code/datums/diseases/transformation.dm [hunk 2]
```diff
@@ -286,6 +286,23 @@
...
+	cure_text = /datum/reagent/water/holywater
+	cures = list(/datum/reagent/water/holywater)
+	agent = "Spectral Curse"
+	desc = "A 'gift' from the spectral realm"
+	stage_prob = 20
+	severity = DISEASE_SEVERITY_BIOHAZARD
+	visibility_flags = 0
+	stage1	= list("Your stomach rumbles.")
+	stage2	= list("Your body feels tired")
+	stage3	= list("<span class='danger'>Your skin starts to look trasnparent</span>", "<span class='danger'>Your limbs begin to lose their shape.</span>")
```
Comment (patpol4) [inline code/datums/diseases/transformation.dm]: trasnparent

File: code/datums/diseases/transformation.dm [hunk 3]
```diff
@@ -286,6 +286,23 @@
...
+	stage4	= list("<span class='danger'>your organs seems to suddenly disapeer.</span>")
```
Comment (patpol4) [inline code/datums/diseases/transformation.dm]: disapeer

**Review by Ktlwjec1 — APPROVED** [review]
map fine

**Review by Amelia0010 — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**